docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -1,3 +1,6 @@
-Mission proficiency level: xref:proficiency_foundational[*{proficiency-foundational}*].
+Mission proficiency level: 
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*Foundational*].]
```

Comment:
Thanks @tradej, nice catch on the last var, sorry I missed that.

Re: swarm guide, yes, thats intentional.  I figured it was a sensible default since we can't know the runtime from the front end without some extra work and ifdef logic.